src/Client/User.php

```diff
@@ -81,4 +81,26 @@ public function deleteUser(string $userId): bool
             throw $this->getExceptionByStatusCode($method, $url, $response);
         }
     }
+
+    public function toggleUser(string $userId, bool $locked): bool
+    {
+        $url = $this->getApiUrl() . 'users/' . urlencode($userId) . '/lock';
+        $headers = $this->getRequestHeaders();
+        $method = 'POST';
+
+        $body = json_encode(
+            [
+                'locked' => $locked,
+            ],
+            JSON_THROW_ON_ERROR
+        );
+
+        $response = $this->getHttpClient()->request($method, $url, $headers, $body);
+
+        if ($response->getStatus() === 200) {
+            return true;
```

Comment:
Why return `true` if you're never going to return `false`? Just return `void` and let us be happy there was no exception.

Reply:
If you search the code we have that in many places like this. The benefit over void is that you can rely on it if it was successful? e.g. `if (someFunction()) {}` ? A void return type would return `null` and you can't be sure it worked?

Reply:
Agreed on sticking with existing code style. Would be better to refactor in next major version.

Reply:
tracking it here https://github.com/btcpayserver/btcpayserver-greenfield-php/issues/99